base16-theme.el

```diff
@@ -87,20 +87,6 @@ the base16-shell code.  If you're using a different terminal
 color scheme, you may want to look for an alternate theme for use
 in the terminal.")
 
-(defun base16-transform-color-key (key)
-  "Transform a given color `KEY' into a theme color.
-
-This function is mostly meant for transforming values based on
-settings and will be for very specific cases."
-  (if (symbolp key)
-      (cond
-       ((string= (symbol-name key) "fringe-bg")
-        (if base16-distinct-fringe-background
-            :base01 :base00))
-       (t
-        (intern (concat ":" (symbol-name key)))))
-    nil))
-
 (defun base16-apply-settings (value)
```

Comment:
I've been trying to think of better ways to do this in general, because I'm not too sure about the overall design in the first place...

What would you think of appending settings-related symbols to the colors plist inside base16-set-faces and removing the code farther in which does translations based on the symbol name? It seems like we're essentially doing a list lookup based on the symbol name anyway.

Reply:
That was one of the ideas I had, too, for the original change but for reasons that escape me now I couldn't make it work. That was likely my lack of understanding of the code but it would be the cleaner solution.